docs/platform-engineers/addon/addon-cue.md

```diff
@@ -188,7 +188,11 @@ output: {
 If you execute the command to enable the addon as follows:
 
 ```shell
-$ vela addon enable <addon-name> clusters=local,cluser1
+$ vela addon enable <addon-name> --clusters=local,cluster1
```

Comment:
does it work? we don't have `--` before.

Reply:
If you don't have `--`:
![image](https://user-images.githubusercontent.com/105624645/203948423-d9875735-d382-4c17-9aac-3cd7da5e7faf.png)

If you have `--` or `"{}"`,it will works.
![image](https://user-images.githubusercontent.com/105624645/203949070-d02bbd4c-e4cc-4038-bb5a-d68e8b1e5829.png)

Reply:
thanks